README.md

```diff
@@ -1,7 +1,19 @@
 # rapidx2j
 
-Node.JS module for converting XML documents into JSON objects. It should be one
-of the fastest converters available. Uses [RapidXML](http://rapidxml.sourceforge.net/).
+Node.JS module for converting XML documents into JSON objects. It is one
+of the fastest parsers available, and outperforms\* all known XML parsers for Node.JS.
```

Comment:
perhaps 'converters' sounds better, since rapidx2j is not only an XML parser, but in fact is an XML to JSON converter?